### ISSUE TITLE
GeoView - UX - Keyboard Accessibility

### ISSUE THREAD

**Issue by sean-eagles** [Canadian-Geospatial-Platform/geo.ca#163]
GeoView users should be able to accomplish tasks using keyboard only.

**Comment by sean-eagles** [comment]
Duplicate of 162.
